packages/components/src/components/hds/app-header/home-link.ts

```diff
@@ -12,21 +12,40 @@ import type { HdsInteractiveSignature } from '../interactive/';
 export interface HdsAppHeaderHomeLinkSignature {
   Args: HdsInteractiveSignature['Args'] & {
     icon: HdsIconSignature['Args']['name'];
+    isIconOnly?: boolean;
     color?: string;
-    ariaLabel: string;
+    text: string;
   };
   Element: HdsInteractiveSignature['Element'];
 }
 
 export default class HdsAppHeaderHomeLink extends Component<HdsAppHeaderHomeLinkSignature> {
-  get ariaLabel(): string {
-    const { ariaLabel } = this.args;
+  get text(): string {
+    const { text } = this.args;
 
     assert(
-      '@ariaLabel for "Hds::AppHeader::HomeLink" ("Logo") must have a valid value',
-      ariaLabel !== undefined
+      '@text for "Hds::AppHeader::HomeLink" must have a valid value',
+      text !== undefined
     );
 
-    return ariaLabel;
+    return text;
+  }
+
+  get icon(): HdsIconSignature['Args']['name'] {
+    const { icon } = this.args;
+
+    assert(
+      '@icon for "Hds::AppHeader::HomeLink" must have a valid value',
+      icon !== undefined
+    );
+
+    return icon;
+  }
+
+  get isIconOnly(): boolean {
+    if (this.icon) {
+      return this.args.isIconOnly ?? false;
+    }
+    return false;
   }
```

Comment:
The Icon is always there, unlike in the `Button` component, so you can simplify this logic.

```
get isIconOnly(): boolean {
  return this.args.isIconOnly ?? false;
}
```